com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,22 @@
+app-id: com.github.TuxPusher
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: tuxpusher
+rename-icon: tuxpusher
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - name: tuxpusher
+    buildsystem: simple
+    build-commands:
+      - cd TuxPusher_SDL;make;make PREFIX=${FLATPAK_DEST} install
```

Comment:
if you add the `subdir: TuxPusher_SDL` property you don't need to `cd` and every build step will be done from that subdirectory.